packages/react/src/components/Forms/Checkbox/index.scss

```diff
@@ -0,0 +1,103 @@
+.c__checkbox__group {
+
+  &__list {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  .c__field__footer {
+    padding: 0.25rem 0 0 calc(2rem);
+  }
+}
+
+.c__checkbox {
+  display: inline-flex;
+  padding: 0.25rem;
+  border-radius: 2px;
+  box-sizing: border-box;
+  border-style: solid;
+  border-color: transparent;
+  // To automatically align the checkbox to the left side.
+  margin-left: -0.25rem;
+
+  &:hover, &:focus-within {
+    background-color: var(--c--theme--colors--greyscale-200);
+  }
+
+  &:focus-within {
+    outline: 0;
+    border-color: var(--c--theme--colors--primary-600);
+  }
+
+  &__container {
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    gap: 0.5rem;
+    color: var(--c--components--forms-checkbox--color);
+  }
+
+  .c__field__footer {
+    padding: 0.25rem 0 0 calc(2rem);
+  }
+
+  &__wrapper {
+    position: relative;
+
+    input {
+      appearance: none;
+      margin: 0;
+      background-color: white;
+      width: 1.5rem;
+      height: 1.5rem;
+      border: 1.5px solid var(--c--components--forms-checkbox--border-color);
+      border-radius: var(--c--components--forms-checkbox--border-radius);
+      display: block;
+
+      &:focus {
+        outline: none;
+      }
+    }
+
+    svg {
+      position: absolute;
+      top: 0;
+      left: 0;
+      color: var(--c--components--forms-checkbox--marks-color);
+
+      &.checkmark {
+        animation: enter var(--c--theme--transitions--duration) var(--c--theme--transitions--ease-out);
+      }
+    }
+  }
+
+
+  @keyframes enter {
+    0% { clip-path: inset(0 100% 0 0); }
+    100% { clip-path: inset(0 0 0 0); }
+  }
```

Comment:
I think we should have a `exit` keyframe to animate checkmark when user unchecks the input.

Reply:
Done 🥵